jobQueue/jobWrapper.js

```diff
@@ -15,6 +15,10 @@
  */
```

Comment:
Maybe export "mapJobResultsToLogObject" and add some tests just sending it mock objects?